README.md

```diff
@@ -1,43 +1,466 @@
+//       "www.example.com/rest/v1": "YOUR_REST_ENDPOINT_HERE"
+//    }
+// }
+
+/*
+ * MyMainExample -- a class that contains only a hello world main method
+ * that defines a rest endpoint.
+ */
+public class MyMainExample {
+  String rest_endpoint;
+
+  public static void main(String[] args){
+    System.out.println("Hello world!");
+    rest_endpoint = "www.example.com/rest/v1"
+  }
+}
+// :replace-end:
+```
+
+Running the following command:
+
+```
+bluehawk copy Main.java -d .
+```
+
+Produces the following output:
+
+`Main.java`:
+
+```java
+/*
+ * Main -- a class that contains only a hello world main method.
+ */
+public class Main {
+  String rest_endpoint;
+
+  public static void main(String[] args){
+    System.out.println("Hello world!");
+    rest_endpoint = "YOUR_REST_ENDPOINT_HERE"
+  }
+}
+```
+
+### Emphasize
+
+The `emphasize` command only applies to [formatted](#format)
+output. When the `--format` flag is not used to generate formatted
+output, `emphasize` is discarded completely. When the `--format` flag
+is specified, Bluehawk highlights all lines marked with `emphasize`
+command in the specified markup language output. `emphasize` makes it
+easier to keep the correct lines highlighted when you update code
+samples, because it calculates the highlighted line numbers for you.
+You can use `replace` as either a block command or a
+line command.
+
+Consider the following file:
+
+`Main.java`:
+
+```java
+public class Main {
+  public static void main(String[] args){
+    // :code-block-start: modulo
+    int dividend = 11;
+    int divisor = 3;
+    int modulus = dividend % divisor; // :emphasize:
+    System.out.println(dividend + " % " + divisor + " = " + modulus);
+    // :code-block-end:
+  }
+}
+// :replace-end:
+```
+
+Running the following command:
+
+```
+bluehawk snip Main.java -d . --format=sphynx-rst
 ```
 
-Try out some of the commands and options:
+Produces the following output:
 
-- `help`: Display helpful information about available commands. May be more complete and 
-  up-to-date than this README.
-- `copy --state <state name>`: Copy the given `state name` version of files to destination.
-  When Bluehawk encounters a `state` command (see below), multiple versions of the source file
-  are spawned. Each version removes any code in state commands that are **not**
-  marked with the corresponding state name. This flag determines which version
-  to eventually write to disk.
-- `snip`: Output snippet files only. Can be combined with `--state`.
-- `list commands`: Display a list of available markup commands.
-- `-d or --destination` defines the output location.
+`Main.codeblock.modulo.java.code-block.rst`:
+
+```rst
+.. code-block:: java
+   :emphasize-lines: 3
+
+   int dividend = 11;
+   int divisor = 3;
+   int modulus = dividend % divisor;
+   System.out.println(dividend + " % " + divisor + " = " + modulus);
+```
+
+### Remove
+
+The `remove` command, also aliased as `hide`, removes the spanned
```

Comment:
I don't think we should encourage people to use the aliases. I'd like to get rid of them.
```suggestion
The `remove` command removes the spanned
```

Reply:
That's why I only mention the aliases once per summary, in an aside. We use `hide` and `code-block` an awful lot in docs-realm so I think we've shown that aliases do matter, in all honesty. I also think we should really really really try to avoid breaking API changes (like removing aliases) in future releases because they just make it harder and harder to understand the tool by eroding existing understanding.

Reply:
Exactly, so let's not mention them here.

This is the benefit of being on a pre-v1 release. Breaking changes are to be expected.

At some point, someone needs to go through the current docs and use the correct name. (Btw, I'm ok with either code-block or snippet, but we should pick one.)